src/include/platform/internal/NFCCommissioningManager.h

```diff
@@ -0,0 +1,113 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    Copyright (c) 2018 Nest Labs, Inc.
```

Comment:
nit: Nest Labs is the copyright holder of that file, was that copied from nest labs project? Also, for the chip project, the copyright date should indicate when the file was introduced to the project, not the date of project creation. For such file it does not matter, but for some patented algorithm then it is clear whether algorithm was open sourced before the patent claim (in which case patenting such algorithm is not possible) or after that (in which case the usage of such code in a retail product might require a patent fee).